### DISCUSSION TITLE
Save parent pedigreeID ID in tree-sequence of individual meta data

### DISCUSSION THREAD

**Discussion by lerch-a** [MesserLab/SLiM#331] [Ideas]
It would be great if the parent pedigreeID could be saved in the tree-sequence, when the keepPedigrees=T in initializeSLiMOptions.
I think the easiest way would be to save the parent pedigreeID in the meta data of the individual like the pedigreeID of the individual.

I would appreciate if this could be integrated in SLiM v4. 

See https://groups.google.com/g/slim-discuss/c/i_BThb7mQDg/m/nXBzQBZCAwAJ for more discussions
Thanks

I'm using SLiM for a mosquito model of a remote island. Currently, I write every mating event into a text file. As mosquito have a lot of offspring this file becomes very large and SLiM simulation is slowed down because of it. However, I'm only interested in the adult mosquitos which I save with sim.treeSeqRememberIndividuals(inds) when they become adults.

**Comment by bhaller** [comment]
Better as an issue, let's discuss this over at https://github.com/MesserLab/SLiM/issues/332.
